react/wbooks/src/app/screens/Home/styles.module.scss

```diff
@@ -4,13 +4,14 @@
 
 .books-container {
   flex-wrap: wrap;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: $navbar-height;
+  margin: $navbar-height auto 0 auto;
   max-width: 935px;
   width: 100%;
 }
 
-.container {
-  background-color: $wild-sand;
+.no-books-message {
+  color: $earls-green;
+  font-size: 32px;
+  margin-top: $navbar-height + 35;
```

Comment:
again with margin-bottom thing, also this result works, i think the result is `margin-top: 100` (100 is hypothetic) and the `px` is missing

Reply:
$navbar-height is 70px. I should put + 35px or + 35?

Reply:
its ok, i realized that navbar height definition countained px, and sass automatically put the px